scripts/kinit_helper

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+function kauth {
+    # If token doesn't exist, create it (will query for password)
+    if ! klist -s
+    then
+	while ! kinit
+	do
```

Comment:
Does this loop just make it so you will be repeatedly queried until you type the password correctly? If so, this is pretty neat

Reply:
Yes. shellcheck recommended using ! in a conditional instead of explicitly checking $?